### DISCUSSION TITLE
Making Public Servers available when bearware.dk is down or having problems

### DISCUSSION THREAD

**Discussion by computertech1012** [BearWare/TeamTalk5#1726] [Ideas]
Hi,
Sometimes, the website bearware.dk has problems, and goes down sometimes, or clients are not able to get the public servers from the list. I was thinking, maybe we could add a way that the clients will save the last time it got the public server list, and so if the server was offline, it could still read the list from which it last got the servers.
For example, let's say the website that hosts the public servers list is down, we would still be able to access the public servers, as long as we have accessed them at least once, and have gotten updates from the public servers list. So basicly it will continue to work as it does now, accept that it will save an offline copy of the list every once in a while as well. Of course, we could also make it an option for people who do not want such a feature.
I think this would be good, because if the website does go down for any reason, we can still at least get to the public servers easily. Also, every time we check the servers, it will update the server list.
So in other words, make the servers save somehow as an offline copy as well, sort of like how if you have an application that is offline, sometimes you can still read your messages until you connect to the server, then you'll get updated messages.
Let me know if I could provide more information and just to clarify this feature would still update the server list, but if it can't, then it will just read from the last update.
Thanks for reading!

**Comment by lbk2907** [comment]
How about for the people who are click on the refresh button? Are it will delete the old copy and refresh the new copy or just update the list?

**Comment by lbk2907** [comment]
Maybe we can add the button to save the official/public/unofficial server list when we want to.

**Comment by bear101** [comment]
If you toggle the "public servers" checkbox then the server list will be refreshed.

**Comment by computertech1012** [comment]
Hi,
Hello @lbk2907 I agree with your idea, to save a copy for offline use.
@bear101 What do you think of this idea? It would be good because then also the offline copy could be used for when the server list is down.
Or we can still use the offline copy feature so that it automaticly saves and offline copy every once in a while. I think it would be better this way and more reliable on the client end.
Thanks for reading!
